django_toolkit/fallbacks/circuit_breaker/circuit_breaker.py

```diff
@@ -107,7 +105,8 @@ def _increase_request_count(self):
         ):
             return
 
-        self.cache.add(
-            self.rule.request_cache_key, 0, self.failure_timeout
-        )
+        self.cache.add(self.rule.request_cache_key, 0, self.failure_timeout)
+        # To calculate the exact percentage, the cache of requests and the
+        # cache of failures must expire at the same time.
+        self.cache.add(self.rule.failure_cache_key, 0, self.failure_timeout)
```

Comment:
Shouldn't it verify the `self.rule.should_increase_failure_count()` before creating this key? As some `Rule`s might not track failures / requests, it may not have the `[failure|request]_cache_key`.